<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Home" failures="1" tests="2" name="Test" time="4.814" errors="0" timestamp="2023-12-03T22:35:24 IST">
  <testcase classname="testng.TestNG1" name="testCase1" time="1.36"/>
  <testcase classname="testng.TestNG1" name="testCase2" time="2.811">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testng.TestNG1.testCase2(TestNG1.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testCase2 -->
</testsuite> <!-- Test -->
